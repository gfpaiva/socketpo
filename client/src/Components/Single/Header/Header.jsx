import React from 'react';

import { Link } from 'react-router-dom';

import './Header.scss';

const Header = () => (
	<div className="single__head">
		<h1 className="single__socketpo"><Link to='/'>SocketPo</Link></h1>
	</div>
);

export default Header;
