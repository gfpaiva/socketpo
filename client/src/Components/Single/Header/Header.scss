.single {
	&__socketpo {
		display: inline-block;
		margin: 0;
	}
}
