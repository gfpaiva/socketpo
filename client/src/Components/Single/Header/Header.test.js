import React from 'react';
import { mount } from 'enzyme';
import { BrowserRouter as Router } from 'react-router-dom';
import Header from './Header';

describe('<Header />', () => {

	it('should mount properly', () => {
		const wrapper = mount(
			<Router>
				<Header />
			</Router>
		);

		expect(wrapper.find('Header')).toMatchSnapshot();
	});
});
