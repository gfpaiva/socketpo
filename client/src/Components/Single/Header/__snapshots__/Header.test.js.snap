// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should mount properly 1`] = `
<Header>
  <div
    className="single__head"
  >
    <h1
      className="single__socketpo"
    >
      <Link
        replace={false}
        to="/"
      >
        <a
          href="/"
          onClick={[Function]}
        >
          SocketPo
        </a>
      </Link>
    </h1>
  </div>
</Header>
`;
